examples/resources/ovirt_disk_from_image/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    ovirt = {
+      source  = "ovirt/ovirt"
+      version = "0.3.0"
```

Comment:
In the [Makefile](https://github.com/oVirt/terraform-provider-ovirt/blob/main/Makefile#L6) the version is still at `0.1.0`. Do we need to bump it up?

Reply:
Yes, eventually when we release it.